packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -63,40 +61,44 @@ const CardSection = ({ heading, cards, theme }) => {
   };
 
   return (
-    <section
-      data-autoid={`${stablePrefix}--card-section`}
-      className={classNames(`${prefix}--card-section`, _setTheme(theme))}>
-      <div className={`${prefix}--card-section__container`}>
-        <div className={`${prefix}--card-section__row`}>
-          <h2 className={`${prefix}--card-section__heading`}>{heading}</h2>
-          <div className={`${prefix}--card-section__cards`} ref={containerRef}>
-            <div
-              className={`${prefix}--card-section__cards__row ${prefix}--row--condensed`}>
-              {cards.map((card, index) => {
-                return (
-                  <div className={`${prefix}--card-section__cards__col`}>
-                    <Card
-                      key={index}
-                      image={card.image}
-                      title={card.heading}
-                      eyebrow={card.eyebrow}
-                      copy={card.copy}
-                      href={card.cta.href}
-                      target={card.cta.target}
-                      icon={ArrowRight20}
-                      type="link"
-                    />
-                  </div>
-                );
-              })}
-            </div>
-          </div>
-        </div>
-      </div>
-    </section>
+    <ContentSection
+      heading={heading}
+      customClassName={classNames(`${prefix}--card-section`, _setTheme(theme))}
+      ref={containerRef}>
+      {_renderCards(cards)}
+    </ContentSection>
   );
 };
 
+/**
+ * Renders the cards based on the CardSection entries
+ *
+ * @param {Array} cards objects array
+ * @returns {*} CardSection JSX objects
+ */
+const _renderCards = cards => (
+  <div
+    className={`${prefix}--card-section__cards__row ${prefix}--row--condensed`}>
+    {cards.map((card, index) => {
+      return (
+        <div className={`${prefix}--card-section__cards__col`}>
+          <Card
+            key={index}
+            image={card.image}
+            title={card.heading}
+            eyebrow={card.eyebrow}
+            copy={card.copy}
+            href={card.cta.href}
+            target={card.cta.target}
+            icon={ArrowRight20}
+            type="link"
+          />
```

Comment:
Keep an eye on @annawen1 pull request. We may see some conflicts here, but we should be able to use a spread here when here changes get merged today.

Reply:
Okay